### PR TITLE
update landing page design

### PR DESCRIPTION
**Pull request by amit-ksh**
- Revamp the whole home page of the application
- Add navbar animation
- Add the new section **text reveal section**

[Preview](https://drive.google.com/file/d/1TWro7pLsskRdPTJIfizQRhLYXByKWLlL/view?usp=sharing)

### CODE REVIEW

**Comment by mrprince88** [comment]
@amit-ksh this looks good. Can you resolve the conflicts?

**Comment by amit-ksh** [comment]
Done!

File: src/app/page.tsx [hunk 1]
```diff
@@ -3,13 +3,16 @@
...
+import TextRevealEffect from "@/components/landing/text-reveal-effect";
 
 export default function App() {
   return (
-    <>
+    <div className="bg-gray-950">
       <Nav />
       <Hero />
+      <TextRevealEffect words="Lorem ipsum dolor sit amet consectetur adipisicing elit. Earum tenetur praesentium ex omnis eos harum non qui minima eveniet laudantium!" />
+
```
Comment (Bhaumik-Tandan) [inline src/app/page.tsx:14]: please replace with some text

**Review by Bhaumik-Tandan — CHANGES_REQUESTED** [review]

**Comment by mrprince88** [comment]
Can you remove the lorem ipsum text and also remove links from header. We dont need these

**Comment by amit-ksh** [comment]
Updated the code as you said @mrprince88

**Pull request merged**